### PR TITLE
Feature/reset password view

### PR DESCRIPTION
**Pull request by Ivan-Marquez**
# Reset Password View

## Change log:
+ Added view engine configuration to express with [Handlebars](http://handlebarsjs.com/)
+ Added reset password view
+ Added front-end / back-end validations for user current password and new password confirmation

## Additional changes:
+ Refactoring functions to handle errors with global error handler
+ Added [Bower](https://bower.io/) to manage front-end dependencies
+ Added nodemon / standard scripts configurations to ignore public directory

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  "homepage": "https://github.com/Ivan-Marquez/stylepoint",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "public/lib",
+    "test",
+    "tests"
+  ],
+  "dependencies": {
```
Comment (kevinpr007) [inline bower.json]: Super Nice!

File: package.json [hunk 2]
```diff
@@ -29,21 +31,30 @@
     "url": "https://github.com/Ivan-Marquez/stylepoint/issues"
   },
   "homepage": "https://github.com/Ivan-Marquez/stylepoint#readme",
+  "standard": {
```
Comment (kevinpr007) [inline package.json]: What's it for?

File: src/config/config.js [hunk 3]
```diff
@@ -12,11 +12,6 @@ switch (ENV) {
       path: `${__dirname}/.env_staging`
     })
     break
-  case 'production':
```
Comment (kevinpr007) [inline src/config/config.js]: Excellent!

File: src/views/layout.hbs [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        </div>
+    </div>
+</div>
+<div class="container body-content" style="margin-top:50px;">
+    {{{body}}}
+    <hr/>
+    <footer class="footer">
+        <div class="container">
+            <div class="copyright pull-right">
+                &copy; {{year}}, <i class="fa fa-code"></i> with <i class="fa fa-heart heart"></i> by Pulsar
```
Comment (kevinpr007) [inline src/views/layout.hbs]: For now it's ok but we need to have a variable to our company name.

Comment (Ivan-Marquez) [inline src/views/layout.hbs]: No prob. I'll add it now if you want.

File: package.json (hunk 2 above)
Comment (Ivan-Marquez) [inline package.json]: To ignore the public directory when we run the standard script.

File: src/views/layout.hbs (hunk 4 above)
Comment (kevinpr007) [inline src/views/layout.hbs]: Don't worry, it's not a big issue at this moment.

**Pull request merged**